src/main/java/net/fabricmc/loader/FabricLoader.java

```diff
@@ -111,23 +113,27 @@ public EnvType getEnvironmentType() {
 	 * @return The game instance's root directory.
 	 */
 	@Override
-	public File getGameDirectory() {
+	public Path getGameDir() {
 		return gameDir;
 	}
 
 	/**
 	 * @return The game instance's configuration directory.
 	 */
 	@Override
-	public File getConfigDirectory() {
```

Comment:
This old method needs to be kept here to ensure compat with really old mods from before we moved to the other interface with the same name in the api package.

Its not ideal but I would remove the default methods from the interface and keep the implimentation here.